public/javascripts/controllers.js

```diff
@@ -1,692 +1,816 @@
 angular.module('uforia')
 
-.controller('appCtrl', function($rootScope){
-  $rootScope.Utils = {
-     keys : Object.keys
-  }
-})
-
-.controller('searchCtrl', function($rootScope, $scope, $http, $modal, $timeout, types) {
-  $scope.queryMatchesCount = 0;
-  $scope.maxResultCount = 0;
-
-  $scope.searchTypes = types;
-  $scope.searchType = $scope.searchTypes[0];
-  $scope.parameters = [{operator: "must", andOr: "And"}];
-  var visualization = {};
-
-  $scope.$watch('viewType', function(newVal, oldVal){
-    if(newVal){
-      removeSVG();
-      changeScripts(function(){
-        if($scope.searchForm.$valid && $scope.queryMatchesCount < $scope.maxResultCount){
-          $scope.search();
+    .controller('appCtrl', function($rootScope){
```

Comment:
I see that GitHub see this as changes though it already existed in previous commits. If this is the case and you are not the one who wrote this code, you can leave it as it is.